### DISCUSSION TITLE
Any project something like Model Engine?

### DISCUSSION THREAD

**Discussion by portlek** [Minestom/Minestom#1189] [Help] [answered]
Is there any project like Model Engine(spigot plugin)?

**Comment by ryleu** [comment]
This isn’t a particularly good answer, because I don’t have the time to go searching myself, but you can look here for a list of all extensions: https://minestom.net/extensions/

**Comment by dev-hydrogen** [comment, marked as answer]
https://github.com/unnamed/hephaestus-engine
